File: Capitulo_2/JavaScript/C-Exercicios-Propostos(pag-50-ate-53)/exercicio-7.js
let prompt = require('prompt-sync')()

let salarioBase = 1500
let nomeCorretor = prompt('Nome do Corretor:')
let qntImoveisVendidos = Number(prompt('Quantidade de imóveis vendidos:'))
let valorTotalVendas = Number(prompt('Valor total em vendas do vendedor:'))

salarioBase = salarioBase + (200 * qntImoveisVendidos) + (5/100 * valorTotalVendas)
console.log(`Salario de ${nomeCorretor}: ${salarioBase}`)

File: Capitulo_2/Python/C_Exercicios_Propostos(pag-50-ate-53)/exercicio-7.py
salario = 1500
nome_corretor = input('Nome do Corretor:')
qnt_imoveis_vendidos = int(input('Quantidade de Imoveis Vendidos:'))
valor_total_vendas = float(input('Valor total em vendas do vendedor:'))

salario = salario + (200 * qnt_imoveis_vendidos) + (5/100 * valor_total_vendas)

print(f'Salario de {nome_corretor}: {salario}')